hack/testing-olm/test-metrics.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
What is this test script good for? How is this different to `test-200-verify-es-metrics-access` and its new golang counterpart?